### ISSUE TITLE
HeidiSQL window goes behind other window after running SQL file

### ISSUE THREAD

**Issue by Beep6581** [HeidiSQL/HeidiSQL#351]
HeidiSQL 9.5.0.5293 in "Windows XP Home Version 2002 Service Pack 3"

When I run an SQL file, often the HeidiSQL window will go behind some other window once the file is done running.

Steps to reproduce:
1. Run `notepad.exe`, maximize it.
2. Run HeidiSQL, click "Open" in the Session Manager, then File > Run SQL File, and run something. My SQL file imports a DB with data, takes a few seconds to run. Once done running, the HeidiSQL window flashes - the Notepad window was visible for a split second, but then HeidiSQL took focus.
3. Run another SQL file. My other SQL file modifies the imported table, takes a few seconds to run. Once done running, this time the HeidiSQL window disappears and the Notepad window is visible (visible but not focused, judging by the window title bar gray color).

Reproducible 90% of the time.

**Comment by ansgarbecker** [comment]
Duplicate of #201
